### ISSUE TITLE
Http2 Support

### ISSUE THREAD

**Issue by LaiZhou** [AdguardTeam/gomitmproxy#8]
Do you have any plans to support http2 proxy?

**Comment by ameshkov** [comment]
Definitely, but this is a rather complicated task and we'll get to it later after we start using gomitmproxy in AdGuard Home.
